src/main/java/net/masterthought/cucumber/util/Status.java

```diff
@@ -13,7 +13,8 @@
     SKIPPED("#88AAFF"),
     PENDING("#FBB907"),
     UNDEFINED("#FBB957"),
-    MISSING("#FBB9A7");
+    MISSING("#FBB9A7"),
+    HIDDEN("");
```

Comment:
This will require a major refactor :)

The view should be handled by the velocity template and the logic by the
java, so we would strip the java methods that wraps html code.

I think I can come up with something for 0.0.26 release.
On May 13, 2015 10:24 PM, "Kingsley Hendrickse" <notifications@github.com>
wrote:

> In src/main/java/net/masterthought/cucumber/util/Status.java
> <https://github.com/masterthought/cucumber-reporting/pull/101#discussion_r30266294>
> :
>
> > @@ -13,7 +13,8 @@
> >      SKIPPED("#88AAFF"),
> >      PENDING("#FBB907"),
> >      UNDEFINED("#FBB957"),
> > -    MISSING("#FBB9A7");
> > +    MISSING("#FBB9A7"),
> > +    HIDDEN("");
>
> At the moment the Java code contains logic and view concerns mixed
> together - long term I would like to separate these out so the Java code is
> view / display agnostic
> … <#14d4ebbfa5dbc5c8_>
> Sent from my iPhone
>  On 13 May 2015, at 20:19, Damian Szczepanik <notifications@github.com>
> wrote: In src/main/java/net/masterthought/cucumber/util/Status.java: > @@
> -13,7 +13,8 @@ > SKIPPED("#88AAFF"), > PENDING("#FBB907"), >
> UNDEFINED("#FBB957"), > - MISSING("#FBB9A7"); > + MISSING("#FBB9A7"), > +
> HIDDEN(""); check with isBackground method so I guess you can make it
> similar but for sure not here — Reply to this email directly or view it on
> GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/masterthought/cucumber-reporting/pull/101/files#r30266294>
> .
>